presto-main/src/test/java/com/facebook/presto/operator/scalar/TestIsNullAnnotation.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.IsNull;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlNullable;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+import io.airlift.slice.Slice;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.type.BigintType.BIGINT;
+import static com.facebook.presto.spi.type.VarcharType.VARCHAR;
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.airlift.slice.Slices.utf8Slice;
+
+public class TestIsNullAnnotation
+        extends AbstractTestFunctions
+{
+    private TestIsNullAnnotation()
+    {
+        registerScalar(getClass());
+    }
+
+    @Description("test @IsNull for one parameter")
+    @ScalarFunction(value = "test_is_null_simple")
```

Comment:
No need for `value` since this is the default inferred from the Java method name